docs/canonicalk8s/snap/reference/troubleshooting.md

```diff
@@ -1,15 +1,15 @@
 # Troubleshooting
 
-This page provides techniques for troubleshooting common {{product}}
-issues.
+If you find any issue while working with {{product}} it is highly
+likely that someone from the community has already faced the same problem.
+In this page you'll find a list of common issues and solutions for them.
 
-Make sure to also check the [troubleshooting how-to guide] for more details on
-how to verify the status of {{product}} services.
+These pages provide techniques for troubleshooting common {{product}}
```

Comment:
I think this line is a duplication of the previous line so we can remove one of them